src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -31,10 +31,7 @@ export interface CommandEventArgs<T extends Slick.SlickData> {
     command: SortDirection;
 }
 
-const ShowFilterText = locConstants.queryResult.showFilter;
```

Comment:
this was saving the value from locConstants the very first time this loaded, and never again.   Since this would run before loc had loaded, these would always be the English strings.